redis/5.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,49 @@
+= Redis Connector 5.2 - Mule 4
+:page-aliases: connectors::redis/redis-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Remote Dictionary Server (Redis Connector) is an open source, in-memory data structure store used for use cases such as caching a user profile, creating a leader board, tracking real time activities on a website, and implementing a spam filtering system. You can use Redis Connector as a message queue, custom object store, and more.
+
+For compatibility information and fixed issues, see the xref:release-notes::connector/redis-connector-release-notes-mule-4.adoc[Redis Connector Release Notes].
```

Comment:
Valkyrie and I just talked about this xref and decided that a better approach is to simply name the RNs and put the link in See Also:
```suggestion
For compatibility information and fixed issues, see Redis Connector Release Notes.
```